lib/jsdom/living/generated/events/EventTarget-impl.js

```diff
@@ -190,12 +190,8 @@ function getListeners(target, type, capturing) {
 
       // Will be falsy for windows that have closed
       if (document) {
-        var implementation = document.implementation;
```

Comment:
I removed it because it broke some existing tests because XMLHttpRequest and FileReader does not really have "inline listener". I put it back but by trying to test if the target is a Node or not.